frontend/public/components/cloud-services/index.tsx

```diff
@@ -112,6 +112,20 @@ export type ClusterServiceVersionResourceKind = {
 
 } & K8sResourceKind;
 
+export type StepResource = {
```

Comment:
Any reason these types are not at the bottom of the file?  Super minimal question, just comparing to patterns across repo, looks like we usually put types at the end.

Reply:
I've been using `index.ts` files for general type definitions and helper functions used in the directory. Also, `eslint` complains about `no-use-before-define`, even though TypeScript doesn't care.

Reply:
Makes sense.

I rather wish we had done a number of component sets like this, vs the file-with-all-the-things.  For example, `/components/builds/` dir with a `index.ts` of helpers like you mentioned, then the `build` and `build-config` files, etc.  

Organizationally, seems to be more future-proof.

Reply:
I have allergy to lots of things in a file :)